### ISSUE TITLE
1.21.1

### ISSUE THREAD

**Issue by frgthyjuierjk** [LambdAurora/LambdaBetterGrass#109]
[from this issue](<https://github.com/LambdAurora/LambdaBetterGrass/issues/100#issuecomment-1993758810>)

Does the mod work on 1.21.1? If not, will there be an update in the near future to tackle it?

**Comment by masuteri** [comment]
The mod appears to be dead, there hasn't been any activity since last year

**Comment by FaultyFunctions** [comment]
So unfortunate because this was one of the better "better grass" mods out there.

**Comment by LambdAurora** [comment]
I do plan on updating this mod.

But currently my focus is LambDynamicLights as I want to first bring it up to better standards, once it's stabilized my focus will be able to shift to my other mods once more.
Most of my mods need a overhaul and re-engineering to be up to a quality I would expect from a similar mod.

Not to mention that now I have a job, this makes my free time much rarer.

**Comment by FaultyFunctions** [comment]
Thank you so much for the response and thank you for planning on updating it. No worries, take your time!

**Comment by SussyBob** [comment]
> So unfortunate because this was one of the better "better grass" mods out there.

agree, this mod is simple but gives tons of aesthetic. Thanks a lot for @LambdAurora for developing this mod!

**Comment by LambdAurora** [comment]
[The mod is now released for Fabric 1.21.1!](https://modrinth.com/mod/lambdabettergrass/version/2.0.0+1.21.1)
